test/config/tslint-custom-rules-with-package-fallback.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rulesDirectory": ["../external/tslint-test-custom-rules/rules"]
```

Comment:
Actually I wanted a test where the path was relative but did not start with `./` or `../`
Consider adding a folder `test/config/relativeRulesDirectory` and change this line to `"rulesDirectory": ["relativeRulesDirectory"]`